docusaurus/docs/get-started/prompts-cli.md

```diff
@@ -64,48 +64,48 @@ Add a [GitHub workflow](/create-a-plugin/develop-a-plugin/set-up-github-workflow
 
 ## CLI commands
 
-Use the CLI for essential tasks of plugin development, employing `npm`, `pnpm`, or `yarn` (for example, `npm run build`).
+Use the CLI for essential tasks of plugin development, employing `npm`, `pnpm`, or `yarn`.
```

Comment:
I think it would be _much_ nicer if we had the npm/yarn/pnpm selector here, failing that, see comment in get-started for a suggestion.

Reply:
Agreed. It was originally available but it removes the titles from the table of contents. I don't really see an issue with that as CLI commands is still there but the PR that rearranged all the Get Started section removed it so I'm guessing @josmperez felt it important to include in the ToC?